src/main/resources/application.properties

```diff
@@ -1,6 +1,8 @@
 spring.application.name=eiffel-intelligence
 #${project.artifactId}
 
+build.version=@project.version@
+
```

Comment:
Build version is not a setting option info.
Can "build.version" be put in a different file, that can handle these kind information?

Reply:
Its ok to keep the version parameter here.